.gitignore

```diff
@@ -1,6 +1,6 @@
 # Emacs
 *~
-docs/v1/build/
+docs/v1/
```

Comment:
I don't think we want docs/v1/ directory in .gitignore, do we?  If I understand the git behavior correctly, making this change will cause _no_ files in docs/v1 or its subdirectories to show up in the output of commands like `git status`, and I think it might also make it impossible to `git commit` files in that directory, unless you first modify the `.gitignore` file.

Reply:
Sorry, you're right